internal/engine/executor.go

```diff
@@ -380,6 +399,7 @@ func (e *Executor) releaseLockAndFlush(
 func logEval(
 	ctx context.Context,
 	inf *entities.EntityInfoWrapper,
+	profileName string,
```

Comment:
This is already part of the evalParams, i.e. `params.Profile.Name`

Reply:
addressed in the new patch